OCD-2084.sql

```diff
@@ -0,0 +1,72 @@
+DROP TABLE IF EXISTS openchpl.participant_gender_statistics;
+
+CREATE TABLE openchpl.participant_gender_statistics
+(
+  	id bigserial NOT NULL,
+  	male_count bigint NOT NULL,
+	female_count bigint NOT NULL,
+  	creation_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_user bigint NOT NULL,
+  	deleted boolean NOT NULL DEFAULT false,
+  CONSTRAINT participant_gender_statistics_pk PRIMARY KEY (id)
+);
+
+CREATE TRIGGER participant_gender_statistics_audit AFTER INSERT OR UPDATE OR DELETE on openchpl.participant_gender_statistics FOR EACH ROW EXECUTE PROCEDURE audit.if_modified_func();
+CREATE TRIGGER participant_gender_statistics_timestamp BEFORE UPDATE on openchpl.participant_gender_statistics FOR EACH ROW EXECUTE PROCEDURE openchpl.update_last_modified_date_column();
+
+DROP TABLE IF EXISTS openchpl.participant_age_statistics;
+
+CREATE TABLE openchpl.participant_age_statistics
+(
+  	id bigserial NOT NULL,
+  	age_count bigint NOT NULL,
+	test_participant_age_id bigint NOT NULL,
+  	creation_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_user bigint NOT NULL,
+  	deleted boolean NOT NULL DEFAULT false,
+  CONSTRAINT participant_age_statistics_pk PRIMARY KEY (id)
+);
+
+CREATE TRIGGER participant_age_statistics_audit AFTER INSERT OR UPDATE OR DELETE on openchpl.participant_age_statistics FOR EACH ROW EXECUTE PROCEDURE audit.if_modified_func();
+CREATE TRIGGER participant_age_statistics_timestamp BEFORE UPDATE on openchpl.participant_age_statistics FOR EACH ROW EXECUTE PROCEDURE openchpl.update_last_modified_date_column();
+
+DROP TABLE IF EXISTS openchpl.participant_education_statistics;
+
+CREATE TABLE openchpl.participant_education_statistics
+(
+  	id bigserial NOT NULL,
+  	education_count bigint NOT NULL,
+	education_type_id bigint NOT NULL,
+  	creation_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_date timestamp without time zone NOT NULL DEFAULT now(),
+  	last_modified_user bigint NOT NULL,
+  	deleted boolean NOT NULL DEFAULT false,
+  CONSTRAINT participant_education_statistics_pk PRIMARY KEY (id)
```

Comment:
Foreign key to education_type?

Reply:
I'll make this change.  I waffled back and forth about this.  Don't remember my logic for not adding it...